devices.js

```diff
@@ -6359,6 +6359,22 @@ const devices = [
                 {'minimumReportInterval': 10, 'reportableChange': 2});
         },
     },
+    {
+        zigbeeModel: ['RS_00.00.02.06TC'],
+        model: 'RS-23ZBS',
+        vendor: 'Climax',
+        description: 'Temperature & humidity sensor',
+        supports: 'Temperature measurements',
+        fromZigbee: [fz.temperature],
+        toZigbee: [],
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
+            await bind(endpoint, coordinatorEndpoint, ['msTemperatureMeasurement']);
```

Comment:
To get humidity measurements you probably have to add `msRelativeHumidity` to this list. Add `await configureReporting.humidity(endpoint);` below and add `fz.humidity` to `fromZigbee`.

Reply:
I'll try that.

Reply:
This is the error I'm getting trying to retrieve the humidity:
```txt
Jun 12 18:56:26 homeassistant-server npm[5523]: zigbee2mqtt:error 2020-06-12 18:56:26: Failed to configure '0x00124b00061c23ce', attempt 1 (Error: ConfigureReporting 0x00124b00061c23ce/1 msRelativeHumidity([{"attribute":"measuredValue","minimumReportInterval":10,"maximumReportInterval":3600,"reportableChange":10}], {"timeout":10000,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) failed (Error: Status 'UNREPORTABLE_ATTRIBUTE')
Jun 12 18:56:26 homeassistant-server npm[5523]:     at Endpoint.<anonymous> (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/controller/model/endpoint.js:329:23)
Jun 12 18:56:26 homeassistant-server npm[5523]:     at Generator.next (<anonymous>)
Jun 12 18:56:26 homeassistant-server npm[5523]:     at fulfilled (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/controller/model/endpoint.js:5:58))
```

 and this is what I see when I console log the device argument:
```txt
Jun 12 18:56:21 homeassistant-server npm[5523]: Device {
Jun 12 18:56:21 homeassistant-server npm[5523]:   ID: 8,
Jun 12 18:56:21 homeassistant-server npm[5523]:   _type: 'EndDevice',
Jun 12 18:56:21 homeassistant-server npm[5523]:   _ieeeAddr: '0x00124b00061c23ce',
Jun 12 18:56:21 homeassistant-server npm[5523]:   _networkAddress: 44370,
Jun 12 18:56:21 homeassistant-server npm[5523]:   _manufacturerID: 0,
Jun 12 18:56:21 homeassistant-server npm[5523]:   _endpoints: [
Jun 12 18:56:21 homeassistant-server npm[5523]:     Endpoint {
Jun 12 18:56:21 homeassistant-server npm[5523]:       ID: 1,
Jun 12 18:56:21 homeassistant-server npm[5523]:       profileID: 260,
Jun 12 18:56:21 homeassistant-server npm[5523]:       deviceID: 770,
Jun 12 18:56:21 homeassistant-server npm[5523]:       inputClusters: [Array],
Jun 12 18:56:21 homeassistant-server npm[5523]:       outputClusters: [Array],
Jun 12 18:56:21 homeassistant-server npm[5523]:       deviceNetworkAddress: 44370,
Jun 12 18:56:21 homeassistant-server npm[5523]:       deviceIeeeAddress: '0x00124b00061c23ce',
Jun 12 18:56:21 homeassistant-server npm[5523]:       clusters: [Object],
Jun 12 18:56:21 homeassistant-server npm[5523]:       _binds: []
Jun 12 18:56:21 homeassistant-server npm[5523]:     }
Jun 12 18:56:21 homeassistant-server npm[5523]:   ],
Jun 12 18:56:21 homeassistant-server npm[5523]:   _manufacturerName: 'ClimaxTechnology',
Jun 12 18:56:21 homeassistant-server npm[5523]:   _powerSource: 'Battery',
Jun 12 18:56:21 homeassistant-server npm[5523]:   _modelID: 'RS_00.00.02.06TC',
Jun 12 18:56:21 homeassistant-server npm[5523]:   _applicationVersion: 12,
Jun 12 18:56:21 homeassistant-server npm[5523]:   _stackVersion: undefined,
Jun 12 18:56:21 homeassistant-server npm[5523]:   _zclVersion: 1,
Jun 12 18:56:21 homeassistant-server npm[5523]:   _hardwareVersion: 1,
Jun 12 18:56:21 homeassistant-server npm[5523]:   _dateCode: '20150331        ',
Jun 12 18:56:21 homeassistant-server npm[5523]:   _softwareBuildID: undefined,
Jun 12 18:56:21 homeassistant-server npm[5523]:   _interviewCompleted: true,
Jun 12 18:56:21 homeassistant-server npm[5523]:   _interviewing: false,
Jun 12 18:56:21 homeassistant-server npm[5523]:   meta: {},
Jun 12 18:56:21 homeassistant-server npm[5523]:   _lastSeen: 1591984581478
Jun 12 18:56:21 homeassistant-server npm[5523]: }
```

It only has two clusters: `msTemperatureMeasurement` and `genBasic` which is only basic information about the device.

Reply:
I don't know why, but I can see the humidity now 🙃

Reply:
I'll update the PRs with the relative humidity support.